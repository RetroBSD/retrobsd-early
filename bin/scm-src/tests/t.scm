123
-123
#o123
#o-123
#b123
#b-123
#x123
#x-123
#d123
#d-123
123.456e12
-123.456e-12
(if 1 2 3)
(if #f 2 3 4)
(cond (#f 2) (#f 1) (else 3))
(eval '(if #f 'x1 'x2 'x3))
(define memq (lambda (obj lst) (cond
    ((null? lst) #f)
    ((eq? obj (car lst)) lst)
    (else (memq obj (cdr lst))) )))
(memq 'b '(a b c d))
