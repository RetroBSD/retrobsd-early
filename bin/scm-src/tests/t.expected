123 --> 123
-123 --> -123
83 --> 83
-83 --> -83
1 --> 1
23 --> 23
-1 --> -1
23 --> 23
291 --> 291
-291 --> -291
123 --> 123
-123 --> -123
1.23456e+14 --> 1.23456e+14
-1.23456e-10 --> -1.23456e-10
(if 1 2 3) --> 2
(if () 2 3 4) --> 4
(cond (() 2) (() 1) (else 3)) --> 3
(eval '(if () 'x1 'x2 'x3)) --> x3
(define memq (lambda (obj lst) (cond ((null? lst) ()) ((eq? obj (car lst)) lst) (else (memq obj (cdr lst)))))) --> <closure-51a5>
(memq 'b '(a b c d)) --> (b c d)
